spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
+
+The fully qualified domain name is secured by a TLS/SSL certificate with an
+optional path. The formal rules describing valid domain name syntax are
+described in [RFC1035], [RFC1123], and [RFC2181]. The fully qualified domain
+name MUST match the common name used in the SSL/TLS certificate, and MUST NOT
+include IP addresses. A port MAY be included and the colon MUST be percent
+encoded to prevent a conflict with paths. Directories and subdirectories MAY
+optionally be included, delimited by colons rather than slashes.
+
+After the fully qualified domain name and optional path in a `did:webs` is a
+final path component, a colon and the AID. To be compatible with `did:web`, the
+AID is "just a path", the final (and perhaps only) path element. The presence of
+the required AID as a path element means that a `did:webs` always has a path,
+and so the "no path" version of a `did:web` that implicitly uses the
+`.well-known` folder is not supported by `did:webs`. Any `did:webs` can be
+expressed as a `did:web` but the inverse is not true--a `did:webs` MUST include
+an AID.
+
+As with `did:web`, the DIDDoc is found by transforming the DID to an
+HTTPS URL as follows:
+
+- Replace `did:webs` with `https://`
+- Replace the "`:`"s in the method-specific identifier with path separators, "'/'"s
+- Convert the optional port percent encoding ("`%3A`"`) to a colon.
+- Append "`/did.json`" to the resulting string.
+
+A GET on that URL MUST return the DIDDoc.
+
+Another transformation produces a second URL that is used to retrieve the KERI KEL:
+
+- Replace the trailing "`/did.json`" with "`/did.kel`".
+
+A GET on that URL MUST return the KEL for the AID in the `did:webs` identifier.
+
+After retrieval, the KEL SHOULD BE processed using KERI rules by anyone
+resolving the DIDDoc to verify it's contents. Further, KERI-aware applications
+MAY use the KEL to make use of additional capabilities enabled by the use of
+KERI. Capabilities beyond the verification of the KEL and DIDDoc are outside of
+the scope of this specification.
+
+The following are some `did:webs` DIDs and their corresponding DIDDoc and KEL
+URLs, based on the examples from the [DID Web Specification], but with the (faked) AID
+(`12124313423525`) added:
+
+- `did:webs:w3c-ccg.github.io:12124313423525`
+  - DIDDoc URL: https://w3c-ccg.github.io/12124313423525/did.json
+  - KEL URL: https://w3c-ccg.github.io/12124313423525/did.kel
+- `did:webs:w3c-ccg.github.io:user:alice:12124313423525`
+  - DIDDoc URL: https://w3c-ccg.github.io/user/alice/12124313423525/did.json
+  - KEL URL: https://w3c-ccg.github.io/user/alice/12124313423525/did.kel
+- `did:webs:example.com%3A3000:user:alice:12124313423525`
+  - DIDDoc URL: https://example.com:3000/user/alice/12124313423525/did.json
+  - KEL URL: https://example.com:3000/user/alice/12124313423525/did.kel
+
+The `did:web` version of the DIDs are the same (minus the `s`) and point
+to the same `did.json` file, but have no knowledge of the `did.kel` file.
```

Comment:
This exposes a problem in making did:web and did:webs such close synonyms: did:web requires the SSL cert and assumes that the website identity = the identity of the DID controller. The security rests on the SSL cert. We break that, substituting stronger security but requiring a different verification procedure. If someone fetches a did:webs but using did:web assumptions, they end up with neither security. This might cause a misunderstanding/bad reasoning. Needs a discussion.